src/frontend/shortlist/.eslintrc.js

```diff
@@ -0,0 +1,24 @@
+/* eslint-env node */
```

Comment:
This file is created automatically when I installed linter locally. I don't think it cause any issue when merging into develop, because it is a well-established file that is meant to set some rules. I do have some caution around it, but I think it is just like a README.MD file that you created whenever you start a new repo. Does no harm and helps my branch work better. I highly doubt it would cause any trouble to other people's codes.